frontend/micro-ui/web/micro-ui-internals/packages/react-components/src/hoc/ResultsTable.js

```diff
@@ -152,7 +152,7 @@ const ResultsTable = ({ tableContainerClass, config,data,isLoading,isFetching,fu
                 globalSearch={config?.enableGlobalSearch ? filterValue : undefined}
                 onSearch={config?.enableGlobalSearch ? searchQuery : undefined}
                 data={searchResult}
-                totalRecords={data?.TotalCount || searchResult?.length }//put total count return from api here
+                totalRecords={data?.count || data?.TotalCount || searchResult?.length }//put total count return from api here
```

Comment:
@prachirane-eGov here pls remove searchResult?.length. It is not required now

Reply:
done